server/services/v1/management.go

```diff
@@ -105,10 +108,26 @@ func (m *managementService) CreateNamespace(ctx context.Context, req *api.Create
 		}
 		code = maxId + 1
 	}
+
+	meta := metadata.NewNamespaceMetadata(code, id, req.GetName())
+
+	// Create a Billing account
+	billingId, err := m.BillingProvider.CreateAccount(ctx, id, req.GetName())
```

Comment:
@JigarJoshi this is how billing is tied to create a namespace.

Once we move to the central user backend then I think there are a few things that we need to think,
- Billing account creation may need to happen on all regional clusters because we need to push metrics to Metronome(or any billing provider)?
- Other things like updating a plan, or modifying any resource may only need to happen at the central backend.

Reply:
@JigarJoshi and this is another area that will be impacted with namespaceId collisions